### DISCUSSION TITLE
Nested Content to Block List migration tool

### DISCUSSION THREAD

**Discussion by Lasher** [umbraco/Umbraco-CMS#14502] [Features and ideas]
Hi,
Most of the content in the Umbraco sites i'm working with, was built with nested content.
Can't see how a manual conversion is going to happen, since there is just too much data to convert.
I Would like to request a migration tool that will convert Nested Content to Block List.

I think that it's a legit request, since Block Lists can behave like Nested Content types
so a migration process should be possible from a structural point of view.

Any assist would be much appreciated
Thanks

**Comment by bergmania** [comment]
Hi Lasher..

Have you checked out a tool like [uSyncMigrations](https://github.com/Jumoo/uSyncMigrations) or [Moriyama Elevate](https://moriyama.co.uk/services/moriyama-elevate/), and most likely also other community tools. Those are excellent in helping with this migration. 

Due to the nature of nested content we don't think we can built a tool in Umbraco that can migrate in all cases, especially because the nested content can contain types Umbraco dont know about.

Sometimes it is also a good idea to consider if the content structure is optimal, before just migrating blindly 😄
